code/modules/mob/dead/crew_manifest.dm

```diff
@@ -0,0 +1,50 @@
+GLOBAL_DATUM_INIT(crew_manifest_tgui, /datum/crew_manifest, new)
+
+/datum/crew_manifest
+
+/datum/crew_manifest/ui_state(mob/user)
+	return GLOB.always_state
+
+/datum/crew_manifest/ui_status(mob/user, datum/ui_state/state)
+	var/static/list/allowed_mobs_typecache = typecacheof(list(/mob/dead, /mob/living/silicon))
+	return (is_type_in_typecache(user, allowed_mobs_typecache) || user.client?.holder) ? UI_INTERACTIVE : UI_CLOSE
+
+/datum/crew_manifest/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "CrewManifest")
+		ui.set_autoupdate(TRUE)
+		ui.open()
+
+/datum/crew_manifest/ui_static_data(mob/user)
+	var/static/list/job_ordering = list_to_assoc_index(GLOB.command_positions + GLOB.engineering_positions + GLOB.supply_positions + (GLOB.nonhuman_positions - "pAI") + GLOB.civilian_positions + GLOB.gimmick_positions + GLOB.medical_positions + GLOB.science_positions + GLOB.security_positions)
+	return list(
+		"command" = list(
+			"jobs" = GLOB.command_positions,
+			"order" = SSjob.chain_of_command
+		),
+		"icons" = GLOB.id_to_hud,
+		"order" = job_ordering
```

Comment:
there should be alternative sort order by hud, so that sorting order isn't fully reliable on job names.
To be honest, I believe sort order by hud icon makes sense more than job names in most standard situations.